### ISSUE TITLE
The response is returned within the array

### ISSUE THREAD

**Issue by romatallinn** [fasten-project/fasten#132]
The response format still (after #100) doesn't exactly match the standard specifications of [JSON API](https://jsonapi.org). There is a minor flow.

The response is returned within an array, rather than by the object itself.

For example, instead of:
```
[
    {
        "id": 16,
        "package_name": "junit:junit",
        "forge": "mvn",
        "project_name": "JUnit",
        "repository": "http://github.com/junit-team/junit/tree/master",
        "created_at": null,
        "version": "4.12"
    }
]
```

Should be:
```
{
   "id": 16,
   "package_name": "junit:junit",
   "forge": "mvn",
   "project_name": "JUnit",
   "repository": "http://github.com/junit-team/junit/tree/master",
   "created_at": null,
   "version": "4.12"
}
```

**Comment by marcomicera** [comment]
Duplicate of https://github.com/fasten-project/fasten/issues/131 (I've been referencing it in the Wiki already a lot of times, so I'm closing this one).

**Comment by MihhailSokolov** [comment]
There are more endpoints to fix (look [here](https://github.com/fasten-project/fasten/wiki/API-endpoints-for-Maven-projects) for `"Only one element, it won't be an array soon."`)
